src/main/java/seedu/address/logic/commands/DisplayCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
+
+/**
+ * Represents a command to display persons whose names contain specified keywords.
+ * A DisplayCommand object corresponds to a user command to display persons based on name keywords.
+ */
+public class DisplayCommand extends Command {
+
+    public static final String COMMAND_WORD = "display";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Displays person whose name contains "
+            + " the specified keyword.\n"
+            + "Parameters: KEYWORD\n"
+            + "Example: " + COMMAND_WORD + " alice";
+
+    private final NameContainsKeywordsPredicate predicate;
+
+    public DisplayCommand(NameContainsKeywordsPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        model.updateFilteredPersonList(predicate);
+
+        if (model.getFilteredPersonList().size() == 0) {
```

Comment:
Can make this neater through this way:
`model.getFilteredPersonList().isEmpty()` or something equivalent